store/transfer_processor.go

```diff
@@ -198,8 +186,16 @@ func (s *transferProcessor) process(ctx context.Context, events []*model.Transfe
 		return err
 	}
 
-	// Insert balance if it's a subscribed account
+	// Insert balance and calculate diff to total balances
+	addrDiff := make(map[common.Address]map[common.Address]*big.Int)
+	allAddrs := append(addrs, newAddrs...)
 	for contractAddr, addrs := range results {
+		// Get last recorded balance for these accounts
+		latestBalances, err := s.getLatestBalances(contractAddr, allAddrs)
+		if err != nil {
+			s.logger.Error("Failed to get previous balances", "len", len(allAddrs), "err", err)
```

Comment:
Can we log message about `contractAddr`?
such as
`s.logger.Error("Failed to get previous balances", "contractAddr", contractAddr.Hex(), "len", len(allAddrs), "err", err)`

Reply:
done. thanks!